apps/main/src/llamalend/features/market-list/cells/MaxLtvCell.tsx

```diff
@@ -0,0 +1,17 @@
+import { LlamaMarket } from '@/llamalend/entities/llama-markets'
+import { formatPercent } from '@/llamalend/format.utils'
+import Typography from '@mui/material/Typography'
+import type { CellContext } from '@tanstack/react-table'
+
+/**
+ * Looked into expanding the LtvCell component to support the max LTV value,
+ * but that cell uses a user position hook which complicates the whole thing.
+ *
+ * The max LTV value comes from prices API, so figured a smaller separate
+ * cell component was the way to go here.
+ */
+export const MaxLtvCell = ({ getValue }: CellContext<LlamaMarket, number>) => (
```

Comment:
This is not specific to LTV at all. It's just a `TypographyCell` or smth
We need to stop creating cells for everything unless necessary

Reply:
Hmm also something to be said. But if we go that way, such a simple one could just be inlined